silk-core/src/main/scala/org/silkframework/dataset/operations/LocalGetProjectFilesOperatorExecutor.scala

```diff
@@ -0,0 +1,60 @@
+package org.silkframework.dataset.operations
+
+import org.silkframework.config.{Task, TaskSpec}
+import org.silkframework.execution.local.{LocalEntities, LocalExecution, LocalExecutor}
+import org.silkframework.execution.typed.{FileEntity, FileEntitySchema, FileType}
+import org.silkframework.execution.{ExecutionReport, ExecutionReportUpdater, ExecutorOutput}
+import org.silkframework.runtime.activity.ActivityContext
+import org.silkframework.runtime.plugin.PluginContext
+
+import scala.collection.mutable
+
+/** Executes the get project files operator. */
+case class LocalGetProjectFilesOperatorExecutor() extends LocalExecutor[GetProjectFilesOperator] {
+
+  override def execute(task: Task[GetProjectFilesOperator],
+                       inputs: Seq[LocalEntities],
+                       output: ExecutorOutput,
+                       execution: LocalExecution,
+                       context: ActivityContext[ExecutionReport])
+                      (implicit pluginContext: PluginContext): Option[LocalEntities] = {
+    val executionReport = GetProjectFilesOperatorExecutionReportUpdater(task, context)
+
+    val fileEntities =
+      for(filePath <- task.data.getFiles()) yield {
+        val file = pluginContext.resources.getInPath(filePath)
```

Comment:
Because of `getInPath` this operator has the same security issue as the 'Add files' operator. But these should be fixed if the `child` method implements the security check.

Reply:
Fixed in the child method.